app/js/invertedIndex.js

```diff
@@ -1,67 +1,106 @@
+/**
```

Comment:
I believe this is a class, so the file name should be `PascalCased` instead of using `camelCasing`